bookkeeper-common/src/main/java/org/apache/bookkeeper/common/util/OrderedExecutor.java

```diff
@@ -380,6 +400,15 @@ protected OrderedExecutor(String baseName, int numThreads, ThreadFactory threadF
             try {
                 threads[idx].submit(() -> {
                     threadIds[idx] = Thread.currentThread().getId();
+
+                    if (enableBusyWait) {
+                        try {
+                            CpuAffinity.acquireCore();
+                        } catch (Throwable t) {
+                            log.warn("Failed to acquire CPU core for thread {}", Thread.currentThread().getName(),
```

Comment:
need some clarifications (comments) here: what happen if we failed to acquire core.